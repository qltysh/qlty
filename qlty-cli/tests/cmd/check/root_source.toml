bin.name = "qlty"
args = ["check", "--all", "--verbose", "--no-cache"]
