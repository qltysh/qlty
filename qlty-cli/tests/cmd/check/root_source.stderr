     [0/1] [..]Planning...  [..]s
     [1/1] [..]Analyzing all targets... 
âœ” No issues
